docker/lwua-ingest/lwua-py/lwua/templates/update_context_lastmod.sparql

```diff
@@ -0,0 +1,34 @@
+{# 
+    This template generates a SPARQL query to update a context in an admin graph.
+    
+    Variables:
+    - registry_of_lastmod_context: The IRI of the registry graph that will keep track of last modification dates.
+    - context: The IRI of the context to update.
+    - lastmod: The new modification date. If provided, the context will be inserted again with this date.
+    
+#}
+
+PREFIX schema: <https://schema.org/>
+DELETE {
+    GRAPH <{{ registry_of_lastmod_context }}> {
+    <{{ context }}> schema:dateModified ?date .
+    }
+}
+INSERT {
+    {% if lastmod %}
+    GRAPH <{{ registry_of_lastmod_context }}> {
+    {#
+        pyrdf2j here is a hack to convert a Python datetime object to an XSD dateTime literal.
+        <{{ context }}> schema:dateModified "{{ lastmod | xsd('dateTime') }} .
```

Comment:
probably needs to be 
`<{{ context }}> schema:dateModified "{{ lastmod | xsd('xsd:datetime') }}`

Reply:
alternatively we might want to create a new rdfj2 version

where we fix the omission of 'xsd:' at the start

two options:
(1) either at https://github.com/vliz-be-opsci/pyrdfj2/blob/c1305d81b48ec3fac69e6e19fb9435023ca6c39a/pyrdfj2/j2_functions.py#L138

we just add `if not type_name.startswith('xsd:'): type_name='xsd:' + type_name` or even `if type_name not in XSD_FMT_TYPE_FN: type_name = 'xsd:' + type_name` 

(2) either at https://github.com/vliz-be-opsci/pyrdfj2/blob/c1305d81b48ec3fac69e6e19fb9435023ca6c39a/pyrdfj2/j2_functions.py#L117C1-L117C16

where we define the dict we can just duplicate the available keys
```python
XSD_FMT_TYPE_FN = {
    "xsd:boolean": xsd_format_boolean,
    "boolean": xsd_format_boolean,
    "xsd:integer": xsd_format_integer,
    "integer": xsd_format_integer,
...
}
```

like option 2 better

Reply:
b'MALFORMED QUERY: Lexical error at line 12, column 106.  Encountered: "\\n" (10), after : "\\"\\\'2023-11-27T10:15:21.485104\\\'^^xsd:datetime ."'

This after the `{{ lastmod | xsd('xsd:datetime') }}` changes.